artemis/src/main/java/tech/pegasys/artemis/cli/deposit/Eth1PrivateKeyOptions.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.cli.deposit;
+
+import java.io.File;
+import picocli.CommandLine.ArgGroup;
+import picocli.CommandLine.Option;
+
+class Eth1PrivateKeyOptions {
+  @Option(
+      names = {"--eth1-private-key"},
+      required = true,
+      paramLabel = "<KEY>",
+      description = "Ethereum 1 private key to use to send transactions")
+  String eth1PrivateKey;
+
+  @ArgGroup(exclusive = false, multiplicity = "1")
+  Eth1EncryptedPrivateKeystoreOptions keystoreOptions;
```

Comment:
nit: since we are getting rid of private as part of describing the eth1 keystore might as well drop it here as well to be consistent. so Eth1EncryptedKeystoreOptions?